src/plugins/proxy/src/compiled_model.hpp

```diff
@@ -38,7 +38,10 @@ class CompiledModel : public ov::ICompiledModel {
     }
 
     ov::Any get_property(const std::string& name) const override {
-        return m_compiled_model->get_property(name);
+        auto property = m_compiled_model->get_property(name);
+        if (!property._so)
+            property._so = m_compiled_model._so;
+        return property;
```

Comment:
in which case property need the so as well?

Reply:
Yes because proxy returns the properties of the hw compiled model.

Reply:
yes I understand, I mean why do the property need the so, generally they are of common types and hardware independent. maybe I'm missing something?

Reply:
Any contains the shared pointer this can be allocated on the plugin side.